static/js/services/changes.js

```diff
@@ -57,7 +60,9 @@ angular.module('inboxServices').factory('Changes',
         });
     };
 
-    watchChanges();
+    DB().info().then(function(info) {
+      lastSeq = info.update_seq;
+    }).then(watchChanges);
```

Comment:
Isn't this equivalent to saying `now`?

I think lastSeq be initialised to `'now'` on init and then updated on change. I think this would fix the issue of api going down and coming back while the webapp is running without the additional unnecessary request on init.